GettyImages.Api/WebHelper.cs

```diff
@@ -122,7 +104,7 @@ internal WebHelper(Credentials credentials, string baseAddress, DelegatingHandle
                     _credentials.ResetAccessToken();
                     using (var retryClient = new HttpClient(await GetHandlersAsync(headerParameters)))
                     {
-                        httpResponse = await retryClient.PostAsync(uri, new FormUrlEncodedContent(queryParameters));
```

Comment:
I think this is a bug as the `UnauthorizedException` retry doesn't make use of `bodyParameter`.  I've left the call as-is in the new "`WithRetryPolicy`" form but filed a bug on the underlying issue here: https://github.com/gettyimages/gettyimages-api_dotnet/issues/27